haas/model.py

```diff
@@ -227,6 +182,31 @@ def get_console(self):
     def get_console_log_filename(self):
         return '/var/run/haas_console_logs/%s.log' % self.ipmi_host
 
+class Obm(AnonModel):
+    """ Obm superclass supporting various drivers 
+        related to out of band management of servers """
+    type = Column(String, nullable=False)
+
+    __mapper_args__ = {
+            'polymorphic_identity': 'new_obmType',
+            'polymorphic_on': type
+            }
+    @staticmethod
+    def validate(kwargs):
```

Comment:
This is pretty clearly copypasta from the Switch class. If it contained actual functionality I'd suggest a mixin, but since it's just docs It might make more sense to just put the docs somewhere else and refer to them in both places.

@gsilvis, @henn, thoughts?